corehq/apps/style/decorators.py

```diff
@@ -1,27 +1,4 @@
 from functools import wraps
-from corehq.apps.style.utils import set_bootstrap_version3
-from crispy_forms.utils import set_template_pack
-
-
-def use_bootstrap3(view_func):
```

Comment:
Maybe you can make this a `noop` instead, so that pages that are currently being worked on with the decorator still work, and we can remove them after.